README.md

```diff
@@ -198,6 +198,15 @@ This is a brief explanation of each:
 * `enable_fullscreen`: An option to allow learners to launch a lab in fullsceen mode,
   in a separate browser window. (Default: `false`)
 
+* `lab_resize_method`: A [display setting](https://guacamole.apache.org/doc/1.5.5/gug/configuring-guacamole.html#rdp-display-settings) for RDP connections to update the RDP server
+  when the width and height of the client display changes. Possible values are:
+
+  * `display-update` *Only supported with RDP 8.1 and higher*. Uses the "Display Update"
+    channel to request the server to change the display size.
```

Comment:
"RDP 8.1" refers to the Windows RDP implementation, which does not map directly to supported features in xrdp. As far as I have been able to make out from https://github.com/neutrinolabs/xrdp/issues/1272, https://github.com/neutrinolabs/xrdp/pull/2891, and https://github.com/neutrinolabs/xrdp/blob/v0.10.2/NEWS.md, GFX support was implemented in xrdp v0.10.0 (2024-05-10), and a resize bug was subsequently fixed in xrdp v0.10.1 (2024-07-31).

What do you think? Should we perhaps change this to:

```suggestion
  * `display-update` *Only supported with Windows RDP 8.1 or xrdp 0.10.1 (and higher)*.
     Uses the "Display Update" channel to request the server to change the display size.
```

Reply:
Yes, sounds good, thanks!